src/utils.ts

```diff
@@ -1,29 +1,22 @@
 import { keccak_256 as keccak256 } from 'js-sha3'
+import { Token } from './types'
 
-const ADDRESS_REGEX = /^0x[0-9a-fA-F]{40}$/
 const EMPTY_ADDRESS = '0x0000000000000000000000000000000000000000'
 const TRUST_WALLET_BASE_URL =
   'https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum'
-const ETHEREUM_LISTS_BASE_URL =
-  'https://raw.githubusercontent.com/ethereum-lists/tokens/master/tokens/eth'
 
-const ETHEREUM_DATA = {
+const ETHEREUM_DATA: Token = {
   name: 'Ethereum',
   symbol: 'ETH',
+  address: '',
+  decimals: 18,
```

Comment:
I added these to align around `Token` but I'm really not sure if this is accurate?